organization/voting.md

```diff
@@ -19,14 +26,47 @@ check conway's law: "Any organization that designs a system will produce a desig
 
 5 seems to not introduce too many established structures
 
+```{note}
+I follow the rationale. Now, should the collective start at 5? If not, what is the timeline to gradually get to this figure? Are processes defined elsewhere robust to size fluctuations?
+```
+
```

Comment:
the collective should start at whichever size is convenient, growth should be organic and not be forced. If there are fewer than 5 members, so be it, the number 5 was proposed as an ideal long-term size.

The whole idea about this is that the collective operates under certain rules and processes, and its members should strictly follow those rules and processes (that need to be clearly and cleanly documented). If at any time, those processes seem insufficient, amendments should be proposed and voted upon.